api/monetization.js

```
@@ -0,0 +1,277 @@
+    if (expectedSig !== parts[2]) return null;
+    const payload = JSON.parse(Buffer.from(parts[1], 'base64url').toString());
+    if (Date.now() / 1000 > payload.exp) return null;
+    return payload;
+  } catch {
+    return null;
+  }
+}
+
+// ─── Tier → Price ID Mapping ──────────────────────────────────────────────────
+const PRICE_TO_TIER = {
+  [process.env.STRIPE_PRICE_PRO_MONTHLY]:    'PRO',
```

ESLint

[error] 'process' is not defined.

(no-undef)